dkron/job.go

```diff
@@ -377,9 +396,21 @@ func (j *Job) GetNext() (time.Time, error) {
 }
 
 func (j *Job) isRunnable(logger *logrus.Entry) bool {
-	if j.Disabled || (j.ExpiresAt.HasValue() && time.Now().After(j.ExpiresAt.Get())) {
+	if j.Disabled {
+		logger.WithField("job", j.Name).
+			Debug("job: Skipping execution because job is disabled")
+		return false
+	}
+
+	if j.StartsAt.HasValue() && time.Now().Before(j.StartsAt.Get()) {
+		logger.WithField("job", j.Name).
+			Debug("job: Skipping execution because job is not started")
```

Comment:
"job is not started" can be confusing: the job didn't start because it hasn't started...?!
I suggest writing "... job not due to start yet." or something to that effect.